def/core.ts

```diff
@@ -264,7 +263,7 @@ export default function (fork: Fork) {
         .bases("Expression")
         .build("operator", "left", "right")
         .field("operator", AssignmentOperator)
-        .field("left", def("Pattern"))
+        .field("left", or(def("Pattern"), def("MemberExpression")))
```

Comment:
The fact that a `MemberExpression` can appear on the left-hand side of an `AssignmentExpression` does not make it a binding pattern.